docs/src/test/scala/docs/http/scaladsl/server/ExceptionHandlerExamplesSpec.scala

```diff
@@ -77,6 +78,7 @@ object MyImplicitExceptionHandler {
   //#implicit-handler-example
 }
 
+@silent("Evaluation of a constant expression results in an arithmetic error")
```

Comment:
Could we put these directly on the expression?

Reply:
I wanted to avoid `@silent` annotations showing up in the documentation

Reply:
Ah, yes, makes sense of course. Alternatively we could change the code to use a variable from the request to trigger the exception which would be more realistic anyway.